scripts/run-service.sh

```diff
@@ -14,6 +14,8 @@ ACCESS_LOG_FILE="-"
 
 if [ "$DJANGO_SETTINGS_MODULE" = "config.settings_stage" ]; then
     ACCESS_LOG_FILE="/var/log/stage-access.log"
+elif [ "$DJANGO_SETTINGS_MODULE" = "config.settings_uat" ]; then
+    ACCESS_LOG_FILE="/var/log/uat-access.log"
```

Comment:
Is this file something we get access to automatically?

Reply:
no, we need to open a ticket with splunk team (I've already opened it)